pkgs/by-name/ni/nixel/package.nix

```diff
@@ -2,22 +2,33 @@
   lib,
   rustPlatform,
   fetchFromGitHub,
+  nix-update-script,
+  clang_16,
   testers,
   nixel,
 }:
-
-rustPlatform.buildRustPackage rec {
+let
+  clang = clang_16;
```

Comment:
A code comment indicating how we know clang 16 is the right version to pin to here would be nice. Would also help someone in the future answer the question of whether that is still the correct version or if the target has moved because of upstream changes.

Reply:
Will refer to https://github.com/NixOS/nixpkgs/issues/331240#issuecomment-2260565324 since I encountered the same error, when using clang18 and 17
but got no problems using clang16 :+1:

Reply:
So I would add a comment to the Nix file here. Something to the effect of "workaround bindgen issue. Fix should be in bindgen version 0.69.2+"